assets/pages/paper-subscription-landing/components/content.jsx

```diff
@@ -121,6 +122,10 @@ const ContentDeliveryFaqBlock = ({ setTabAction }: {setTabAction: typeof setTab}
   );
 };
 
+function trackedLink(href: string, text: string, onClick: Function) {
```

Comment:
could this be a react element instead?

```suggestion
function TrackedLink({href, text, onClick}:{href: string, children: Node, onClick: Function}) {
```

and then you can call it like
```
<TrackedLink
    href='https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions',
    onClick={()=>{sendClickedEvent('paper_subscription_collection_page-subscription_faq_link')}}
>Subscriptions FAQs</TrackedLink>
```

There's no real advantage to this it's just a bit more react-y in line with the rest of the code